Server/mods/deathmatch/logic/CAccount.cpp

```diff
@@ -289,6 +289,15 @@ bool CAccount::RemoveSerial(const SString& strSerial)
     return false;
 }
 
+//
+// Replace the serial number for a specific account
+//
+void CAccount::SetAccountSerial(const SString strSerial)
```

Comment:
SString is no longer used, please use std::string instead

Reply:
adjusted 62ebf06